### PR TITLE
Use `IntersectionObserver` for lazy loading breach images.

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Uses `IntersectionObserver` to lazy load breach images on `/breaches`.
- Adds `IntersectionObserver` polyfill for older browser versions, Opera, and all but the very latest version of Safari.
- Loads polyfill into top-level pages (`/breaches` and `/security-tips`). We'll need IntersectionObserver for the fast-follow redesigned Security Tips page.
- Removes some old, unused, or unnecessary CSS from `all-breaches.css` and fixes an indentation nit in `all-breaches.hbs`.

### CODE REVIEW

File: template-helpers/hbs-helpers.js [hunk 1]
```diff
@@ -162,6 +162,16 @@ function breachMath(lValue, operator = null, rValue = null) {
 }
 
 
+function IntersectionObserverPolyfill(args) {
+  const topLevel = (args.data.exphbs.view === "top-level-page");
+  if (topLevel) {
+    /* eslint-disable-next-line quotes */
+    return `<script type="text/javascript" src="/dist/intersection-observer.min.js"></script>`;
+  }
+}
```
Comment (pdehaan) [inline template-helpers/hbs-helpers.js]: Does this need any other `return` statement?

I see we have this below (views/partials/imports.hbs#9), but wanted to make sure we don't get into a weird case where we aren't trying to write `null` or `undefined` into the DOM:

```hbs
{{{ IntersectionObserverPolyfill }}}
```

**Comment by pdehaan** [comment]
Ref: "Logos on /scan endpoint aren't lazy loaded #1046"

File: public/js/all-breaches.js [hunk 2]
```diff
@@ -84,7 +54,7 @@ function makeBreaches(breachArray, breachCardWrapper) {
     makeBreachInfoSpans("breach-title", breach.Title, wrapper);
 
     // added date
-    makeBreachInfoSpans("breach-key", breach.String.AddedDate, wrapper);
+    makeBreachInfoSpans("breach-key", breach.AddedDate, wrapper);
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply public/js/all-breaches.js]: Sorry I'm just getting back to these comments - I appreciate you going through this! This is addressed in #1070.

**Comment by lesleyjanenorton** [comment]
Closing this because it's now pretty stale. Would like to prioritize this, and the lazy loading of all breach images, once we get the loading of `/breaches` in order.

**Pull request closed without merging**